channels/candidate-4.4.yaml

```diff
@@ -1,5 +1,7 @@
 name: candidate-4.4
 versions:
+# No 4.3.16 because it was rejected by the release controller
```

Comment:
this position will not sort into ART's logic, which prepends 4.(y-1) and appends 4.(y).  So the expected order with that is:

4.3.17
4.3.16
4.3.15
...
4.4.0-rc.0
4.4.0-rc.1
...